src/openrct2/rct2/S6Importer.cpp

```diff
@@ -670,7 +670,7 @@ IParkImporter * ParkImporter::CreateS6(IObjectRepository * objectRepository, IOb
 
 extern "C"
 {
-    ParkLoadResult * game_load_sv6_path(const char * path)
+    ParkLoadResult * load_from_sv6(const char * path)
```

Comment:
Wouldn't it be better to make these functions proper C++ functions and have the extern C functions only call the real ones?